exercises/practice/palindrome-products/.articles/is-palindrome-check/snippet.md

```diff
@@ -0,0 +1,4 @@
+private static bool IsPalindrome(int number)
+{
+    /* implementation details */
+}
```

Comment:
I think we probably want something else here. I can come up with two things:

- A summary of the performance of the various methods
- A snippet of the most interesting implementation

Reply:
I have no started [a post on the forum](https://forum.exercism.org/t/article-snippets/5272) to get wider feedback on the best approach.

Reply:
I've posted there

Reply:
This snippet is the only thing left to do I think. What do you think about a tiny bit of text for the snippet?

Reply:
I agree, that's the last thing left. I've spent a lot of time thinking about it and cannot find anything that really works in my head. The performance benchmark output doesn't give any value or information. I cannot think of a text that would work in scenario where we have

1. short text - the snippet
2. title
3. blurb - a short text

It breaks the flow we are used from newspapers, books, blogposts all around us. It's normally

1. something graphical to capture our attention
2. a short catchy title
3. a brief description to help you decide if you want to click it

The best I can think of is something like this
```
bool IsPalindrome(int number)
{
    /* Let's look how to implement it! */
}
```

It shows the article is about the `IsPalindrome` implementation, it explains what you will get from clicking the link, it will look like code sa be a bit more 'graphical', and not favour one implementation over the other or pre-empt what is in the article. 

That's the best I can come up with, but I'm open to suggestions, or examples from other articles or tracks. Please help, @ErikSchierboom

Reply:
```
bool IsPalindrome(int number)
{
    /* Let's implement it! */
}
```

Reply:
Let's just go with that

Reply:
OK, let's try it and see how it looks. We can always change it. 
I have added the backticks to wrap around the code as you said in a forum that it is just a mardown text. I hope that's right.